main/main.cpp

```diff
@@ -1329,7 +1306,26 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 	{
 		window_orientation = DisplayServer::ScreenOrientation(int(GLOBAL_DEF_BASIC("display/window/handheld/orientation", DisplayServer::ScreenOrientation::SCREEN_LANDSCAPE)));
 	}
-
+	{
+		String vsync_mode = GLOBAL_DEF("display/window/vsync/vsync_mode", "Enabled");
+
+		if (vsync_mode == "Disabled") {
+			print_line("<<Disabled");
+			window_vsync_mode = DisplayServer::VSYNC_DISABLED;
+		} else if (vsync_mode == "Enabled") {
+			print_line("<<On");
+			window_vsync_mode = DisplayServer::VSYNC_ENABLED;
+		} else if (vsync_mode == "Adaptive") {
+			print_line("<<Adaptive");
+			window_vsync_mode = DisplayServer::VSYNC_ADAPTIVE;
+		} else if (vsync_mode == "Mailbox") {
+			print_line("<<Mailbox");
+			window_vsync_mode = DisplayServer::VSYNC_MAILBOX;
+		} else {
+			print_line("v-sync mode unknown.");
+			window_vsync_mode = DisplayServer::VSYNC_ENABLED;
+		}
```

Comment:
It's probably better to use `print_verbose` (need some code rearrangement, since `verbose` flag is set on the line 1344).

Reply:
Oh, forgot to remove those print_line statements. They were just for debugging, so i removed the first four and replaced the last one with WARN_PRINT.